packages/typegpu/src/data/vector.ts

```diff
@@ -326,6 +325,7 @@ function makeVecSchema<TValue, S extends number | boolean>(
         `${type}(${args.map((v) => v.value).join(', ')})`,
         vecTypeToConstructor[type],
       ),
+    'construct',
```

Comment:
change all ''construct's to the name of how the constructor is used. i mean like vec4f, mat4x4f... or at least vec and mat if that's not possible